README.rst

```diff
@@ -1,3 +1,23 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
```

Comment:
This is why I don't like badges: they suck for users with visual impairments.

How about:

```
:alt: latest version of waitress on PyPI
```

Set it and forget it. No need to update for each release.